aioes/transport.py

```diff
@@ -46,8 +46,9 @@ class Transport:
 
     def __init__(self, endpoints, *,
                  sniffer_interval=None, sniffer_timeout=0.1, max_retries=3,
-                 loop, verify_ssl=True):
+                 loop, verify_ssl=True, connector=None):
         self._loop = loop
+        self._connector = connector
```

Comment:
I think here in `Transport` it should be `connector_factory`,
otherwise a single `connector` will be shared between different `Connections` (`ClientSession`s)
and when some connection will get dropped underlying ClientSession will close the shared connector
making other connections broken as well.
I think it could look like this:
```python
connector_factory=lambda: None
```